### PR TITLE
chore: bump versions

### PR DESCRIPTION
**Pull request by dvilelaf**
* Updates the repo to use the latest version of the frameworks

### CODE REVIEW

File: .github/workflows/main_workflow.yml [hunk 1]
```diff
@@ -145,46 +142,123 @@ jobs:
...
+
           # install Tendermint
           wget -O tendermint.tar.gz https://github.com/tendermint/tendermint/releases/download/v0.34.19/tendermint_0.34.19_linux_amd64.tar.gz
           tar -xf tendermint.tar.gz
           sudo mv tendermint /usr/local/bin/tendermint
+
           # pull node image needed for the integration tests
+          docker pull node:16.7.0
           docker pull trufflesuite/ganache:beta
-          docker pull valory/autonolas-registries:latest
```
Comment (angrybayblade) [inline .github/workflows/main_workflow.yml:183]: This is required by default

File: .github/workflows/main_workflow.yml [hunk 2]
```diff
@@ -145,46 +142,123 @@ jobs:
...
-          docker pull valory/safe-contract-net:latest
-      - name: Unit tests
+          docker pull valory/autonomous-fund-contracts:latest
```
Comment (angrybayblade) [inline .github/workflows/main_workflow.yml]: This is not

File: .github/workflows/main_workflow.yml [hunk 3]
```diff
@@ -229,9 +303,9 @@ jobs:
           sudo mv tendermint /usr/local/bin/tendermint
 
           # pull node image needed for the integration tests
-          docker pull trufflesuite/ganache:beta
-          docker pull valory/autonolas-registries:latest
-          docker pull valory/safe-contract-net:latest
+          docker pull node:16.7.0
+
+          docker pull valory/autonomous-fund-contracts:latest
```
Comment (angrybayblade) [inline .github/workflows/main_workflow.yml]: Same, this a template repo it should not contain anything that is app specific

File: Makefile [hunk 4]
```diff
@@ -56,7 +56,7 @@ formatters:
 # darglint: docstring linter
 .PHONY: code-checks
 code-checks:
-	tox -p -e black-check -e isort-check -e flake8 -e mypy -e pylint -e vulture -e darglint
```
Comment (angrybayblade) [inline Makefile]: Why removed vulture?

File: .github/workflows/main_workflow.yml (hunk 1 above)
Comment (dvilelaf) [reply .github/workflows/main_workflow.yml:183]: All this issues you correctly pointed come from my last commit, where I tried to fix the worflow error overwriting the current with the one from the autonomous fund. Reverted now.

File: Makefile [hunk 5]
```diff
@@ -72,7 +72,6 @@ security:
 generators:
 	tox -e abci-docstrings
 	tox -e fix-copyright
-	autonomy hash all
```
Comment (0xArdi) [inline Makefile:75]: Why?

File: scripts/check_copyright.py [hunk 6]
```diff
@@ -311,6 +311,8 @@ def main() -> None:
     python_files = filter(
         lambda x: x not in exclude_files,
         itertools.chain(
+            Path("packages", "valory", "skills", "score_read_abci").glob("**/*.py"),
+            Path("packages", "valory", "skills", "score_write_abci").glob("**/*.py"),
```
Comment (0xArdi) [inline scripts/check_copyright.py]: why?

File: Makefile (hunk 5 above)
Comment (dvilelaf) [reply Makefile:75]: It is deprecated:

```
autonomy hash all
`autonomy hash all` command has been deprecated and will be removed on v2.0.0, please use `autonomy packages lock` command to perform package hash updates
Traceback (most recent call last):
  File "/home/david/.local/share/virtualenvs/dev-template-KOqW1M5f/lib/python3.10/site-packages/autonomy/cli/helpers/ipfs_hash.py", line 84, in update_hashes
    packages[0] = packages[0] + list(map(to_package_id, SCAFFOLD_PACKAGES))
IndexError: list index out of range
make: *** [Makefile:75: generators] Error 1

```

File: scripts/check_copyright.py (hunk 6 above)
Comment (dvilelaf) [reply scripts/check_copyright.py]: Oops, removing

**Review by 0xArdi — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**